addons/purchase/models/ir_actions_report.py

```diff
@@ -0,0 +1,54 @@
+import io
+
+from odoo import models
+from odoo.tools.pdf import OdooPdfFileReader, OdooPdfFileWriter
+
+
+class IrActionsReport(models.Model):
+    _inherit = 'ir.actions.report'
+
+    def _render_qweb_pdf_prepare_streams(self, report_ref, data, res_ids=None):
+        # EXTENDS base
+        collected_streams = super()._render_qweb_pdf_prepare_streams(report_ref, data, res_ids=res_ids)
+
+        if collected_streams \
+                and res_ids \
+                and len(res_ids) == 1 \
```

Comment:
Isn't it possible to download multiple purchase order at once? Then you wouldn't have the xml attachment.

Reply:
It's possible, but how should we include multiple XML files in the same PDF then (from a functional perspective I mean)?

Reply:
I don't know since each stream is merged, you could also have multiple xml files but we can keep it like as it is now with only one file